istiofiles/enduser-authentication-jwt.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "customerjwt"
+  namespace: "tutorial"
+spec:
+  targets:
+  - name: customer
+  origins:
+  - jwt:
+      issuer: "http://keycloak-tutorial.192.168.64.87.nip.io/auth/realms/istio"
+      jwksUri: "http://keycloak-tutorial.192.168.64.87.nip.io/auth/realms/istio/protocol/openid-connect/certs"
```

Comment:
why are these urls hardcoded ?? it will change from machine to machine right?

Reply:
For token generation and so far it is easier. Notice that I have already warned user here https://github.com/redhat-developer-demos/istio-tutorial/pull/137/files#diff-976019af37449dd398b7c178b914aa86R91